### PR TITLE
[staging/elasticsearch] Fix README.md broken links

### PR DESCRIPTION
**Pull request by RochesterinNYC**
- Fixes broken links and images

### CODE REVIEW

File: staging/volumes/scaleio/README.md [hunk 1]
```diff
@@ -2,16 +2,7 @@
...
-     width="25" height="25">
-<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
-     width="25" height="25">
+**WARNING** **WARNING** **WARNING** **WARNING** **WARNING**
```
Comment (ahmetb) [inline staging/volumes/scaleio/README.md]: can you remove this, all the way to line 21?

**Comment by RochesterinNYC** [comment]
Addressed the comment.

**Pull request merged**